projects/example-app/src/app/auth/reducers/index.ts

```diff
@@ -27,11 +29,11 @@ export const selectAuthStatusState = createSelector(
   selectAuthState,
   (state: AuthState) => state.status
 );
+export const getUser = createSelector(selectAuthStatusState, fromAuth.getUser);
 export const getLoggedIn = createSelector(
-  selectAuthStatusState,
-  fromAuth.getLoggedIn
+  getUser,
+  (user: User | null) => !!user
```

Comment:
The typings here aren't needed, it already knows that `user` is `User | null`.

Reply:
Ok, I'll change this. Thanks

Reply:
You can also remove the `User` import now.

Reply:
so true :+1: